force-app/main/fia/classes/KafkaFiaCaseHandler.cls

```diff
@@ -53,6 +59,8 @@ public with sharing class KafkaFiaCaseHandler extends KafkaMessageProcessor impl
     }
 
     private Account convertToSObject(KafkaFiaCase_Model model, Map<String, Account> accountMap) {
+            Map<String, String> navIdentMap = getNavIdentMap();
```

Comment:
Her er det SOQL spørring i en for loop. getNavIdentMap må flyttes ut av convertToSObject metoden.

Reply:
ok